### ISSUE TITLE
Line wrapping in examples

### ISSUE THREAD

**Issue by mattgarrish** [w3c/tr-design#282]
Another request from the epub accessibility horizontal review was to enable line wrapping in markup examples to meet the reflow requirement of WCAG 2.1.

I can again do this manually using `break-spaces` or `pre-wrap`, but wondering if this should be part of the default styling for the `<pre><code>` blocks? (Or maybe it could get applied when the page gets zoomed?)

**Comment by marcoscaceres** [comment]
Transferred to tr-design repo...

**Comment by fantasai** [comment]
The problem with making this the default is that sometimes line breaks are semantically important to the example. So the stylesheets apply `overflow: auto` to allow overflowing text to be seen, but they don't actually allow wrapping because that could distort the example.

Individual specs can add rules to enable wrapping, or we could add a pre.wrap { overflow-wrap: break-word } rule to base.css to make it easier to enable wrapping; but I wouldn't want to enable it by default on all pre elements because that can break some specs.

**Comment by plehegar** [comment]
+1 to @fantasai. In addition, this would be a breaking change if we were modifying the default.

If we had a pre.wrap CSS class, would this be of enough interest? Neither Respec nor Bikeshed contain such a style rule so far, which tells me this isn't used  significantly.

**Comment by mattgarrish** [comment]
> If we had a pre.wrap CSS class, would this be of enough interest?

As an individual rule it's a bit pointless. I don't want to add it to the hundreds of examples across the specs I work on, and have to keep remembering to add it on every time we add a new one.

Could you do something like a global body.example-wrap class, so that all pre tags in the body get wrapped?

That would be better than having to inline a rule manually, and would also presumably allow it to be documented so that others can be made aware of this issue.
